modules/javafx.controls/src/main/resources/com/sun/javafx/scene/control/skin/resources/controls.properties

```diff
@@ -144,3 +144,10 @@ Dialog.warning.header=Warning
 
 Dialog.confirm.title=Confirmation
 Dialog.confirm.header=Confirmation
+
+### Charts ###
+XYChart.series.accessibleText={0} {1} is {2} {3} is {4}
```

Comment:
> Do we provide a hint to the translators on how these strings need to be translated, like giving an example of rendered text?

And that is why i did not do any "Google translate" translation in this PR - i will raise a new bug to translate the new strings where i will describe their meaning and hints on how to translate. Because even if that will be me who is doing the translation i would like to give a reviewer more information on what and how i translate it.

Reply:
one way to do it is to describe the expected result in the comment immediately preceding the string.  For example:

```
# "the current execution is complete"
Class.runText=The run is run.
```

Also, for testing purposes, we could consider pseudolocalization
https://en.wikipedia.org/wiki/Pseudolocalization

Better discuss this with Naoto Sato.

Reply:
Sorry for accidentally editing your comment. I think I restored it correctly.

Reply:
> one way to do it is to describe the expected result in the comment immediately preceding the string.
> ...
> 
> Also, for testing purposes, we could consider pseudolocalization

These are good suggestions that should be captured in the follow-up issue that Alex said he will file.

Localization and testing is out of scope for this PR. Alex did the minimum needed to correct the fact that the string wasn't internationalized at all, which makes sense only because he had to modify the string as part of this PR (even then, I would have been OK with no i18n changes in this PR, leaving them for a separate issue as well).

> Better discuss this with Naoto Sato.

Good idea.

Reply:
Definitely out of scope for this PR, I just wanted to mention this.
@azuev-java please link the follow up JBS ticket for localization.
Thank you!

Reply:
Still, I would recommend adding a comment (since now is the time when we know all the context, and in the future we might forget or lose the context), may be something like

```
# {seriesName} {xAxisName} is {currentX} {yAxisName} is {currentY}
XYChart.series.accessibleText={0} {1} is {2} {3} is {4}
```

Reply:
> Still, I would recommend adding a comment (since now is the time when we know all the context, and in the future we might forget or lose the context), may be something like

Good idea, let me do it for both strings with parameters

Reply:
sorry to hold up the review - I'll speak to Naoto Sato tomorrow (2/3/23 @ 11:00) regarding the requirements for translation hints, and either approve or let you know what they expect us to do.

Reply:
OK. Presuming that the comment Alex already left is fine, or just needs a slight format change, then that's good. If it's more than that, it would be better to do it in a follow-up issue.

Reply:
The comment is good.
The idea of giving translators hints in the form of comments like this was very warmly welcomed by our i18n lead Naoto Sato.  So I think we should follow this approach for every new string we add in jfx (and jdk).
The goal is to give the translators more context, by either giving an example of resulting text or by describing the parameters like Alex did in this case (or both).